templates/community-viz/dist/index.html

```diff
@@ -1,10 +1,19 @@
 <!doctype html>
 <html>
   <head>
-    <title>Getting Started</title>
+    <title>Community Viz</title>
+    <style>
+      body {
+        font-family: Roboto;
```

Comment:
Did you add the css somewhere for this to be loaded from google fonts? It's probably cached on your system (and maybe available in chrome), but I don't think it's available everywhere without adding it in. https://fonts.google.com/specimen/Roboto